src/payments/services/payment.service.ts

```
@@ -258,6 +258,14 @@ export class PaymentService {
         );
     }
 
+    let failedWebhookReason: string | null = null;
+    if (webhookEvent.status === 'failed') {
```

ESLint

[error] Replace `'failed'` with `"failed"`

(prettier/prettier)